### PR TITLE
use strato_common in whiteboxtest

### PR DESCRIPTION
**Pull request by eyalsn-stratoscale**

### CODE REVIEW

**Comment by shimshon-stratoscale** [comment]
Problem - this repo is public, but strato-common isn't.

**Comment by eyalsn-stratoscale** [comment]
@shimshon-stratoscale 
what do you mean?

File: py/strato/whiteboxtest/runner/main.py [hunk 1]
```diff
@@ -12,6 +8,10 @@
 import re
 import os
 import subprocess
+
+from strato_common.log import configurelogging
+from strato.whiteboxtest.runner import config
+configurelogging.configureLogging('whiteboxtest.runner', forceDirectory=config.TEST_LOGS_DIR)  # noqa
```
Comment (eyal-stratoscale) [inline py/strato/whiteboxtest/runner/main.py]: configure logging should be at the top of the file, after importing it.

**Comment by eyal-stratoscale** [comment]
beside small note, ship it.
btw, you sure this works with upseto?

File: py/strato/whiteboxtest/runner/main.py (hunk 1 above)
Comment (eyalsn-stratoscale) [inline py/strato/whiteboxtest/runner/main.py]: configure logging needs to be at the TOP, true, but after the built-ins import

Comment (eyal-stratoscale) [inline py/strato/whiteboxtest/runner/main.py]: why?

On Sun, Jan 31, 2016 at 11:16 AM, eyalsn-stratoscale <
notifications@github.com> wrote:

> In py/strato/whiteboxtest/runner/main.py
> <https://github.com/Stratoscale/pywhiteboxtest/pull/3#discussion_r51357536>
> :
>
> > @@ -12,6 +8,10 @@
> >  import re
> >  import os
> >  import subprocess
> > +
> > +from strato_common.log import configurelogging
> > +from strato.whiteboxtest.runner import config
> > +configurelogging.configureLogging('whiteboxtest.runner', forceDirectory=config.TEST_LOGS_DIR)  # noqa
>
> configure logging needs to be at the TOP, true, but after the built-ins
> import
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Stratoscale/pywhiteboxtest/pull/3/files#r51357536>.
>



-- 


Web <http://www.stratoscale.com/> | Blog <http://www.stratoscale.com/blog/>
 | Twitter <https://twitter.com/Stratoscale> | Google+
<https://plus.google.com/u/1/b/108421603458396133912/108421603458396133912/posts>
 | Linkedin <https://www.linkedin.com/company/stratoscale>

**Comment by eyalsn-stratoscale** [comment]
I added to strato_kv this hash in upseto.manifest so currently its only us who uses it.
all the rest who are not using strato-common can use older versions in their upseto

File: py/strato/whiteboxtest/runner/main.py (hunk 1 above)
Comment (eyalsn-stratoscale) [inline py/strato/whiteboxtest/runner/main.py]: @eyal-stratoscale 
https://www.python.org/dev/peps/pep-0008/#imports

Comment (eyal-stratoscale) [inline py/strato/whiteboxtest/runner/main.py]: https://github.com/Stratoscale/strato-common/blob/master/strato_common/log/configurelogging.py#L21

On Sun, Jan 31, 2016 at 11:21 AM, eyalsn-stratoscale <
notifications@github.com> wrote:

> In py/strato/whiteboxtest/runner/main.py
> <https://github.com/Stratoscale/pywhiteboxtest/pull/3#discussion_r51357598>
> :
>
> > @@ -12,6 +8,10 @@
> >  import re
> >  import os
> >  import subprocess
> > +
> > +from strato_common.log import configurelogging
> > +from strato.whiteboxtest.runner import config
> > +configurelogging.configureLogging('whiteboxtest.runner', forceDirectory=config.TEST_LOGS_DIR)  # noqa
>
> @eyal-stratoscale <https://github.com/eyal-stratoscale>
> https://www.python.org/dev/peps/pep-0008/#imports
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Stratoscale/pywhiteboxtest/pull/3/files#r51357598>.
>



-- 


Web <http://www.stratoscale.com/> | Blog <http://www.stratoscale.com/blog/>
 | Twitter <https://twitter.com/Stratoscale> | Google+
<https://plus.google.com/u/1/b/108421603458396133912/108421603458396133912/posts>
 | Linkedin <https://www.linkedin.com/company/stratoscale>

**Pull request merged**